app/urls.py

```diff
@@ -0,0 +1,34 @@
+from django.urls import path, include
+from .views import (
+    GoogleSignInView,
+    LogoutView,
+    UpdateInstaHandle,
+)
+
+from .admin_views import (
+    AdminSignupView,
+    AdminLoginView,
+    AdminLogoutView,
+    QuestionView,
+)
+
+from .user_QA_views import (
+    DailyQAView,
+    WeeklyQAView,
+)
+from rest_framework import routers
+
+router = routers.DefaultRouter()
+
+
+urlpatterns = [
+    path("login/", GoogleSignInView.as_view()),
+    path("logout/", LogoutView.as_view()),
+    path("update_insta_handle/", UpdateInstaHandle.as_view()),
+    path("daily_question/", DailyQAView.as_view()),
```

Comment:
Dont use underscore in the URL, it is bad practice. Instead use a base url. For example `/admin/signup` instead of `/admin_signup`